src/BenchmarkDotNet/Configs/DebugConfig.cs

```diff
@@ -65,7 +65,20 @@ public abstract class DebugConfig : IConfig
         public IOrderer Orderer => DefaultOrderer.Instance;
         public SummaryStyle SummaryStyle => SummaryStyle.Default;
         public ConfigUnionRule UnionRule => ConfigUnionRule.Union;
-        public string ArtifactsPath => Path.Combine(Directory.GetCurrentDirectory(), "BenchmarkDotNet.Artifacts");
+
+        public string ArtifactsPath
+        {
+            get
+            {
+                var root = Directory.GetCurrentDirectory();
+                if (root == "/")
```

Comment:
Is this the best way to check if we are running on Android? How about:

```suggestion
                if (RuntimeInformation.IsOSPlatform(OSPlatform.Create("ANDROID")))
```



@jonathanpeppers could you please add a comment here that explains why we need this check (it's not obvious)

Reply:
This is the same code as here: https://github.com/dotnet/BenchmarkDotNet/blob/92474e8870244bec76393ee59debd7126fe3bc13/src/BenchmarkDotNet/Configs/DefaultConfig.cs#L80-L91

My thought was that if the current directory is `/`, it's not going to be writeable on any platform.

Let me see if the `IsOSPlatform` check works for me, though.

Reply:
@adamsitnik neither of these checks actually work on the current Xamarin:
```csharp
RuntimeInformation.IsOSPlatform(OSPlatform.Create("ANDROID"))
RuntimeInformation.IsOSPlatform(OSPlatform.Create("Android"))
```
I found some discussion [here](https://gitter.im/xamarin/xamarin-android?at=5bcd64996e5a401c2dadfb0b).

Hopefully, `OSPlatform.Android` will exist in .NET 6, but we probably can't use until then.

I updated BDN's `RuntimeInformation` instead.